addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -32,11 +32,11 @@ const ActivityMenu = Widget.extend({
     //--------------------------------------------------
     // Private
     //--------------------------------------------------
+
     /**
-     * Make RPC and get current user's activity details
      * @private
      */
-    async _getActivityData() {
+    async _updateActivityPreview() {
```

Comment:
Don't see adapt of overridden `_getActivityData` in modules `calendar` and `note`

Reply:
I'm not sure which commit this is referring to but data are now provided from models so override is fields/convertData/python format.